modules/RTC/TraceablePeerConnection.js

```diff
@@ -2571,6 +2573,27 @@ TraceablePeerConnection.prototype.setSenderVideoConstraints = function(frameHeig
     if (!localVideoTrack || localVideoTrack.isMuted()) {
         return Promise.resolve();
     }
+
+    const prevPromise = this._lastVideoSenderUpdatePromise;
+    const nextPromise = Promise.allSettled([ prevPromise ])
```

Comment:
You no longer need to extract prevPromise, you can just use the instance variable.

Reply:
Right, corrected that one.